Source/WebCore/history/CachedFrame.cpp

```diff
@@ -280,8 +281,9 @@ void CachedFrame::destroy()
 
     document->protectedWindow()->willDestroyCachedFrame();
 
-    Ref frame = m_view->frame();
-    if (!m_isMainFrame && m_view->frame().page()) {
+    RefPtr view = m_view;
+    Ref frame = view->frame();
+    if (!m_isMainFrame && view->frame().page()) {
```

Comment:
Existing issue, but still good to fix:
```suggestion
    if (!m_isMainFrame && frame->page()) {
```